miasm2/core/graph.py

```diff
@@ -27,6 +27,14 @@ def add_node(self, n):
         self._nodes_to[n] = []
         self._nodes_from[n] = []
 
+    def del_node(self, n):
```

Comment:
Can you use `node` instead of  `n`, as in `get_all_parents` ?
And add a (tiny) docstring ?